### PR TITLE
Something descriptive

### PR DESCRIPTION
**Pull request by EthanBurdett**

### CODE REVIEW

**Review by ashlyndevries — APPROVED** [review]
LGTM

File: desktop.ini [hunk 1]
```diff
@@ -0,0 +1,2 @@
+[.ShellClassInfo]
+LocalizedResourceName=@wiki,0
```
Comment (ashlyndevries) [inline desktop.ini]: Why this here??

**Review by ashlyndevries — CHANGES_REQUESTED** [review]
Fix it! Please :) Peace & love

**Review by EthanBurdett — COMMENTED** [review]
Hey uhhhhhhhhhghhghhhhhhhhhh Ashlyn? Its ready.......

**Review by ashlyndevries — APPROVED** [review]
LOTM

**Review by ashlyndevries — APPROVED** [review]
LGTM

**Pull request merged**